pilot/pkg/features/telemetry.go

```diff
@@ -51,6 +51,10 @@ var (
 		"If true, pilot will add metadata exchange filters, which will be consumed by telemetry filter.",
 	).Get()
 
+	MetadataExchangeAdditionalLabels = env.Register("PILOT_MX_ADDITIONAL_LABELS", "",
```

Comment:
I don't love this API - making users add strings to the pilot env isn't very user friendly

Reply:
suggested by [john](https://docs.google.com/document/d/1NPl9TyBadejvcOfsTO6rWz9quru4zU6WQmAzQX6iphI/edit?disco=AAABafms1Jc), and I agree with him this's better than my proposal, as this's not part of prometheus provider.